### PR TITLE
convert `switch` to `match`

### PR DESCRIPTION
**Pull request by ximex**
Use the new `match` control statement where possible (https://www.php.net/manual/en/control-structures.match.php)

also nice that we could safe nearly half of the code lines

### CODE REVIEW

File: src/Exceptions/Handlers/LegacyDevelopment.php [hunk 1]
```diff
@@ -88,4 +69,19 @@ protected function formatStackTrace($stackTrace)
 
         return $return;
     }
+
+    /**
+     * @param mixed $arg
+     * @return string
+     */
+    private function getDisplayNameOfValue(mixed $arg): string
```
Comment (MyIgel) [inline src/Exceptions/Handlers/LegacyDevelopment.php:77]: This function is not covered by tests / excluded

**Review by MyIgel — CHANGES_REQUESTED** [review]
That change looks good so far but reduces test coverage:

File: src/Exceptions/Handlers/LegacyDevelopment.php (hunk 1 above)
Comment (MyIgel) [reply src/Exceptions/Handlers/LegacyDevelopment.php:77]: Bspw.
```diff
diff --git a/src/Exceptions/Handlers/LegacyDevelopment.php b/src/Exceptions/Handlers/LegacyDevelopment.php
index 8e32e24a..b1c381a1 100644
--- a/src/Exceptions/Handlers/LegacyDevelopment.php
+++ b/src/Exceptions/Handlers/LegacyDevelopment.php
@@ -80,7 +80,7 @@ class LegacyDevelopment extends Legacy
             'string', 'integer', 'double' => (string)$arg,
             'boolean'  => $arg ? 'true' : 'false',
             'object'   => get_class($arg),
-            'resource' => get_resource_type($arg),
+            'resource' => get_resource_type($arg), // @codeCoverageIgnore
             default    => gettype($arg),
         };
     }
diff --git a/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php b/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php
index bd0c6990..42e81bfe 100644
--- a/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php
+++ b/tests/Unit/Exceptions/Handlers/LegacyDevelopmentTest.php
@@ -11,8 +11,9 @@ use PHPUnit\Framework\TestCase;
 class LegacyDevelopmentTest extends TestCase
 {
     /**
-     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::formatStackTrace()
-     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::render()
+     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::formatStackTrace
+     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::render
+     * @covers \Engelsystem\Exceptions\Handlers\LegacyDevelopment::getDisplayNameOfValue
      */
     public function testRender()
     {
```

Comment (ximex) [reply src/Exceptions/Handlers/LegacyDevelopment.php:69]: added comments

**Pull request merged**